### ISSUE TITLE
Shartuul Event Does Not Work

### ISSUE THREAD

**Issue by testaccnt125** [Atlantiss/NetherwingBugtracker#9119]
[//]: # (Enclose links to things related to the bug using http://wowhead.com or any other TBC database.)
[//]: # (You can use screenshot ingame to visual the issue.)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Azuremyst Isle] Red Snapper - Very Tasty!)
[//]: # ([NPC] Magistrix Erona)
[//]: # ([Spell][Mage] Fireball)
[//]: # ([Npc][Drop] Ghostclaw Lynx)
[//]: # ([Web] Armory doesnt work)
[//]: # (To find server revision type `.server info` command in the game chat.)

**Description**: When you use Crystalforged Darkrune at the Shartuul event crystal orb, nothing happens. I lost my first one, so I submitted a ticket. Pelto said it was reset and should work, so I tried again after getting another one. Nothing happened again and I lost my 2nd darkrune.

**Current behaviour**: As soon as I'm done casting Crystalforged Darkrune, I lose my darkrune and I don't take control of the pet.

**Expected behaviour**:  As soon as I'm done casting Crystalforged Darkrune, I should be taking control of the felguard pet and start the Shartuul event.

**Server Revision**: I am on 3.3.5 client.

**Comment by Godmatik** [comment]
duplicate

https://github.com/Atlantiss/NetherwingBugtracker/issues/8669

entire event needs to be rescripted

**Comment by testaccnt125** [comment]
Hi, I checked that ticket and my issue doesn't seem like a duplicate since that user can at least finish the event despite the bugs; I can't even start the event in the first place, and I just lose my crystalforged darkrune.

I also see that the ticket is from 2023... Will this ever be addressed?

**Comment by Godmatik** [comment]
It's because the trigger for it gets broken, also needs to be fixed.

only fixed after server restart or a GM resetting it

**Comment by testaccnt125** [comment]
Checking in to see status of Shartuul event. Crysis said he'd fix it over the weekend, but last I checked with him on Monday, all I got was "Idk." Is anyone else working on it?

**Comment by Godmatik** [comment]
Crysis can only reset the trigger.
Wolfenstein is one who needs to fix it.

You can DM him about this and reference this thread - but he may have other priorities at the moment.

**Comment by testaccnt125** [comment]
Update on this: I saw another druid (obviously) from opposite faction actually initiate the event (meaning the square boundaries is active), but his character (as in the druid) was still visible and targetable from my end even though the felguard said "xyz's pet" (which is not supposed to be, tauren should have disappeared while he was controlling the pet for the duration of his event). He wasn't moving though (neither his felguard nor his tauren). And I didn't bother to sit there and wait while they were both still.

So after I did some dailies in the area and came back, he was gone and square boundaries were cleared. So I tried using one of my crystalforged darkrunes, but same thing happened as my previous report--I didn't take control of felguard and I lost my darkrune again.

Maybe he was using 2.4.3, I don't know. There's been discrepancies in some bugs happening on 3.3.5, but not 2.4.3.
